src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets

```diff
@@ -114,6 +115,8 @@ Copyright (c) .NET Foundation. All rights reserved.
         <_Parameter1>%(AssemblyMetadata.Identity)</_Parameter1>
         <_Parameter2>%(AssemblyMetadata.Value)</_Parameter2>
       </AssemblyAttribute>
+      <AssemblyAttribute Include="System.Runtime.Versioning.RequiresPreviewFeaturesAttribute" Condition="'$(EnablePreviewFeatures)' == 'true' and '$(GeneratePreviewFeaturesAttribute)' == 'true'">
```

Comment:
I'm not super familiar with MSBuild or the sdk repo. If this is the wrong spot to make these changes, let me know!